canonical-web-and-design/maas.io
# Move app to Flask

## Done

Migrate the app from Django to Flask.

## QA

- Make sure all the site works fine, including versioned static CSS files and homepage RSS feeds.
- Make sure redirects still work fine

Fixes https://github.com/canonical-web-and-design/base-squad/issues/588
